src/templates/parts/screen-cell.hbs

```diff
@@ -29,6 +29,12 @@
       {{/select}}
     </select>
 
+    {{else}}
+
+    <b style="width: 100%;">
+      {{ GMSCR-journal this.entityUuid }}
```

Comment:
try this, might just work:
```suggestion
      {{ lookup @root/entityOptions this.entityUuid }}
```

Reply:
Thanks, very narrowminded from my end. Will see how to make it generic to all entities.

Reply:
All of them should have a name after all.